core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -167,9 +165,7 @@ oppia.controller('StateResponses', [
       if (!outcome) {
         return false;
       }
-      var hasFeedback = outcome.feedback.some(function(feedbackItem) {
-        return Boolean(feedbackItem);
-      });
+      var hasFeedback = outcome.feedback != null;
```

Comment:
isEmpty(). Seems like a different condition than the above, btw?

Also, alternatively, for this stuff it might be better to add an outcome.hasFeedback() or outcome.hasNonemptyFeedback() method. Make sure to test these new methods.

Reply:
fixed